x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/action.ts

```diff
@@ -27,14 +27,8 @@ interface UserPaginatedHostList {
   payload: HostListPagination;
 }
 
-// Why is FakeActionWithNoPayload here, see: https://github.com/elastic/endpoint-app-team/issues/273
-interface FakeActionWithNoPayload {
-  type: 'fakeActionWithNoPayLoad';
-}
-
 export type HostAction =
   | ServerReturnedHostList
   | ServerReturnedHostDetails
   | ServerFailedToReturnHostDetails
-  | UserPaginatedHostList
-  | FakeActionWithNoPayload;
+  | UserPaginatedHostList;
```

Comment:
what helped us get around this issue?

Reply:
I'm not sure what the issue was to begin with. The link in the old comment just says 'No description provided.'